compiler/src/iree/compiler/Codegen/SPIRV/Passes.td

```diff
@@ -112,9 +112,9 @@ def SPIRVTileToCooperativeOps : Pass<
     "mlir::iree_compiler::createSPIRVTileToCooperativeOpsPass()";
 }
 
-def SPIRVVectorize : Pass<"iree-spirv-vectorize", "func::FuncOp"> {
+def SPIRVVectorLowering : Pass<"iree-spirv-vector-lowering", "func::FuncOp"> {
   let summary = "Vectorize Linalg ops with buffer semantics";
-  let constructor = "mlir::iree_compiler::createSPIRVVectorizePass()";
+  let constructor = "mlir::iree_compiler::createSPIRVVectorLoweringPass()";
```

Comment:
We should probably remove these constructors in the future and let tablegem autogenerate all of this code. But this is as a side note only, let's keep things as close to NFC as possible in this PR.

Reply:
yes we should. I split up the TD file to be per backend to do that (someday). I